view/frontend/web/js/view/payment/method-renderer/omise-offsite-installment-method.js

```diff
@@ -45,27 +45,6 @@ define(
                 return this;
             },
 
-            /**
-             * Format Price
-             * 
-             * @param {float} amount - Amount to be formatted
-             * @return {string}
-             */
-            getFormattedAmount: function (amount) {
-                return priceUtils.formatPrice(amount, quote.getPriceFormat());
```

Comment:
Should we also remove `Magento_Checkout/js/model/quote`, `Magento_Catalog/js/price-utils` modules at the beginning of this file?

Reply:
Yup, quote and priceUtils are no longer used

Reply:
👍